storage/namespace.go

```diff
@@ -129,12 +129,21 @@ func (n *dbNamespace) QueryRaw(
 		if err != nil {
 			return nil, err
 		}
-		res.MergeInPlace(shardRes)
+		if res == nil {
+			res = shardRes
+		} else {
+			if err := res.MergeInPlace(shardRes); err != nil {
```

Comment:
`q.NewRawResults().MergeInPlace(shardRes)` results in `merging two raw results with different order by field types` error since the other has an order field while the former doesn't